src/testing.ts

```diff
@@ -69,6 +69,9 @@ const expectedResults = new Map<string, ResultType>([
   ['example_contracts/mutableReferences/memory', 'Success'],
   ['example_contracts/mutableReferences/mutableReferences', 'NotSupportedYet'],
   ['example_contracts/mutableReferences/scalarStorage', 'Success'],
+  ['example_contracts/namedArgs/constructor', 'NotSupportedYet'],
+  ['example_contracts/namedArgs/events_and_errors', 'Success'],
+  ['example_contracts/namedArgs/function', 'Success'],
```

Comment:
These are good example contracts, but could we also have some behaviour tests for the cases which are currently supported, to verify that functions get called with the right arguments in the right order after the transform